charts/k8s-monitoring/templates/hooks/validate-configuration.yaml

```diff
@@ -47,6 +47,8 @@ spec:
       - |
         echo Validating Grafana Alloy config file
         if ! alloy fmt /etc/alloy/config.alloy > /dev/null; then
+          echo Generated Config with Errors
```

Comment:
Can we change this message to say something more like "Grafana Alloy config file has syntax errors"
Similarly, after the "alloy run" check, and before echoing the output, can you add "Grafana Alloy config file has errors"?